### PR TITLE
Easy fix: Rename to  #683

### PR DESCRIPTION
**Pull request by kristinakarnitskaya**
Fixes #\<issue number\>.

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue above.

Note that we are unlikely to accept pull requests that add features without prior discussion. The best way to propose a feature is to open an issue first and discuss your ideas there before implementing them.

Always follow the [contribution guidelines](https://github.com/codebuddiesdotorg/codebuddies/blob/master/contributing.md) when submitting a pull request.

### CODE REVIEW

File: i18n/en.i18n.json [hunk 1]
```diff
@@ -138,7 +138,7 @@
   "members": "Members",
   "learnings": "Progress Log",
   "links": "Links",
-  "resources": "Resources",
+  "links": "Links",
```
Comment (stain88) [inline i18n/en.i18n.json]: Does links need to be in here twice?

Comment (kristinakarnitskaya) [reply i18n/en.i18n.json]: You're right. Will fix it

File: client/templates/study_groups/single_study_group.html [hunk 2]
```diff
@@ -80,7 +80,7 @@
             <li role="presentation"><a href="#hangouts" aria-controls="hangouts" role="tab" data-toggle="tab">{{_ 'hangouts'}}</a></li>
             <li role="presentation"><a href="#members" aria-controls="members" role="tab" data-toggle="tab">{{_ 'members'}} ({{ members.length }})</a></li>
             <li role="presentation"><a href="#learnings" aria-controls="learnings" role="tab" data-toggle="tab">{{_ 'learnings'}} ({{ learningsCount }})</a></li>
-            <li role="presentation"><a href="#resources" aria-controls="resources" role="tab" data-toggle="tab">{{_ 'resources'}} ({{ resourcesCount }})</a></li>
+            <li role="presentation"><a href="#resources" aria-controls="resources" role="tab" data-toggle="tab">{{_ 'links'}} ({{ resourcesCount }})</a></li>
```
Comment (kristinakarnitskaya) [inline client/templates/study_groups/single_study_group.html]: Shall I change all the attributes/links from 'resources' to 'links' or just the name of the link?

**Comment by lpatmo** [comment]
Thanks for the PR @kristinakarnitskaya and thanks for reviewing, @stain88!

**Pull request merged**